integration/csi/go.sum

```diff
@@ -0,0 +1,481 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Is this file necessary?

Reply:
It's auto generated, we should not change this file